pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -193,6 +194,12 @@ func metricsTimerTask(ctx *zedagentContext, handleChannel chan interface{}) {
 			ctx.ps.CheckMaxTimeTopic(wdName, "publishMetrics", start,
 				warningTime, errorTime)
 
+			locConfig := ctx.getconfigCtx.locConfig
+			if locConfig != nil {
+				// Publish all info by timer only for LOC
+				triggerPublishAllInfo(ctx, LOCDest)
+			}
```

Comment:
At some point in time we might want a separate setable timer value for the periodic info to LOC, but we can worry about that later I think.

Reply:
Does this triggerPublishAllInfo cause the location to be publishes as well? Then we should document that where I asked about the location publishing above.

Reply:
Since we already have separate timers for location publishing, I would prefer it those were respected and location wasn't published alongside info or metrics.

Reply:
I did not change the default behavior. If I did - this is a bug. So metrics timer is abused (yes) and used for publishing location info only for the LOC. `locationTimerTask` publishes location from the timer only for the controller and LPS (as it was before).